app/models/admin.rb

```diff
@@ -0,0 +1,2 @@
+class Admin < User
```

Comment:
admin/normal_user should not be subclass of User. It should be a boolean column in user table. If more roles are required then the field should be ENUM.

Reply:
wasn't I supposed to use Single Table Inheritance?